### PR TITLE
removed outdated step

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10032

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md [hunk 1]
```diff
@@ -349,7 +349,7 @@ Follow these steps to run the PowerShell script:
 With the hardware ID captured in a file, prepare your Virtual Machine for Windows Autopilot deployment by resetting it back to OOBE.
 
 On the Virtual Machine, go to **Settings > Update & Security > Recovery** and select **Get started** under **Reset this PC**.
-Select **Remove everything** and **Just remove my files**. If you're asked **How would you like to reinstall Windows**, select Local reinstall. Finally, select **Reset**.
+Select **Remove everything**, then, on the **How would you like to reinstall Windows**, select Local reinstall. Finally, select **Reset**.
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:352]:
```suggestion
Select **Remove everything**, then, on **How would you like to reinstall Windows**, select **Local reinstall**. Finally, select **Reset**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MaratMussabekov, kindly commit the minor suggested changes. Many thanks.

File: windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md (hunk 1 above)
Comment (Lovygoogle) [reply windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:352]: </b>
  </font>
    </span>
       <br />
         <span style="font-family: arial">
           <font size="5">
<b>وذكر المؤلفون نماذج من كتمان الأسرار وهى :</b>
           </font>
         </span>
       <br />
     <span style="font-family: arial">
  <font size="5">
<b>&quot;نماذج من كتمان السر</b>
  </font>
    </span>
      <br />
        <span style="font-family: arial">
          <font size="5">
            <b>1

Comment (Lovygoogle) [reply windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:352]: - [ ]> </b>
>   </font>
>     </span>
>        <br />
>          <span style="font-family: arial">
>            <font size="5">
> <b>وذكر المؤلفون نماذج من كتمان الأسرار وهى :</b>
>            </font>
>          </span>
>        <br />
>      <span style="font-family: arial">
>   <font size="5">
> <b>&quot;نماذج من كتمان السر</b>
>   </font>
>     </span>
>       <br />
>         <span style="font-family: arial">
>           <font size="5">
>             <b>1

Comment (Lovygoogle) [reply windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:352]: <iframe src="https://www.google.com/maps/d/embed?mid=1QzkPZ1ne4kuvPFjmbI57w1E3wCU&hl=ar&ehbc=2E312F" width="640" height="480"></iframe>
> > </b>
> >   </font>
> >     </span>
> >        <br />
> >          <span style="font-family: arial">
> >            <font size="5">
> > <b>
</b>
> >            </font>
> >          </span>
> >        <br />
> >      <span style="font-family: arial">
> >   <font size="5">
> > <b>&quot;
</b>
> >   </font>
> >     </span>
> >       <br />
> >         <span style="font-family: arial">
> >           <font size="5">
> >             <b>1
> 
>

Comment (Lovygoogle) [reply windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:352]: Select **Remove everything**, then, on the **How would you like to reinstall Windows**, select Local reinstall. Finally, select **Reset**.
Select **Remove everything**, then, on **How would you like to reinstall Windows**, select **Local reinstall**. Finally, select **Reset**.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**